### DISCUSSION TITLE
Feedback: Query parameters

### DISCUSSION THREAD

**Discussion by smoothdev35** [mswjs/mswjs.io#428] [Q&A] [answered]
- [Open page](https://mswjs.io/docs/recipes/query-parameters/)

## Feedback

Could you give an example of how to read the path parameters as well ?

**Comment by smoothdev35** [comment, marked as answer]
Found it, thanks.

**Reply by kettanaito** [reply]
For posterity: https://mswjs.io/docs/network-behavior/rest#reading-path-parameters
